sqlglot/parser.py

```diff
@@ -588,15 +595,15 @@ class Parser(metaclass=_Parser):
         TokenType.ARROW: lambda self, expressions: self.expression(
             exp.Lambda,
             this=self._replace_lambda(
-                self._parse_conjunction(),
+                self._parse_op(),
```

Comment:
_parse_expression is already taken.

Here are some other ideas:
_parse_operation
_parse_operator

Reply:
_parse_connector is nice too, yeah

Reply:
But I guess the intent to is to abstract the name a little bit, since so much other code calls this. And most of the code that calls this doesn't actually care about parsing AND,OR, etc - they just want an expression

Reply:
ah ok that's fine then

Reply:
> But I guess the intent to is to abstract the name a little bit, since so much other code calls this. And most of the code that calls this doesn't actually care about parsing AND,OR, etc - they just want an expression

I believe the naming scheme should preserve the information of what "level of precedence" we're parsing with a method, because it ends up getting too abstract and methods can be misused. I've been noticing this pattern over time with method names like `_parse_field` etc - it's not clear what syntax they're able to consume.

Reply:
then we can change to connective or connector since we already use that word